python/louis/__init__.py.in

```diff
@@ -36,28 +37,22 @@ function for information about how liblouis searches for these tables.
 @author: Leonard de Ruijter (Babbage B.V.) <alderuijter@gmail.com>
 """
 
-from __future__ import unicode_literals
-from ctypes import *
+from ctypes import (
```

Comment:
`from ctypes import *` is better, otherwise Linux compatibility is broken 😭:

```
[andreabush@arch-vm ~]$ python -c "from ctypes import c_ushort, WINFUNCTYPE, CFUNCTYPE, windll, cdll, c_char_p, c_char
, c_int, POINTER, byref, create_string_buffer"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'WINFUNCTYPE' from 'ctypes' (/usr/lib/python3.8/ctypes/__init__.py)
```

Reply:
Ah, good catch. Unfortunately that causes linter issues. I don't insist on a succesful lint, but it would help for debugging purposes.

Reply:
Ah, OK. I suggest the following:

```
from ctypes import (
    c_ushort, CFUNCTYPE, cdll,
    c_char_p, c_char, c_int,
    POINTER, byref, create_string_buffer)
try: # Native win32
    from ctypes import WINFUNCTYPE, windll
    _loader, _functype = windll, WINFUNCTYPE
except ImportError: # Unix/Cygwin
    _loader, _functype = cdll, CFUNCTYPE```